src/components/a11y/a11y.js

```diff
@@ -111,11 +131,43 @@ const A11y = {
   },
   init() {
     const swiper = this;
+    const params = swiper.params.a11y;
 
     swiper.$el.append(swiper.a11y.liveRegion);
 
+    // Container
+    const $containerEl = swiper.$el;
+    if (params.containerRoleDescriptionMessage) {
+      swiper.a11y.addElRoleDescription($containerEl, params.containerRoleDescriptionMessage);
+    }
+    if (params.containerMessage) {
+      swiper.a11y.addElLabel($containerEl, params.containerMessage);
+    }
+
+    // Wrapper
+    const $wrapperEl = swiper.$wrapperEl;
+    const randomWrapperId = `swiper-wrapper-${swiper.a11y.getRandomNumber(16)}`;
```

Comment:
I would suggest to reuse existing wrapper ID if it is there, something like:
```
const wapperId = $wapperEl.attr('id') || `swiper-wrapper-${swiper.a11y.getRandomNumber(16)}`;
```
Otherwise people may complain if their custom ID (if the have it for some reason) is overwritten

Reply:
Thanks for your kind and fast reaction.
You are right, I changed the part for the wrapperId as you suggested and added the docs for the new parameters to TypeScript definitions.